pro/src/pages/Home/Venues/Venue.tsx

```diff
@@ -135,6 +136,18 @@ const Venue = ({
   )
   const hasNewOfferCreationJourney = useNewOfferCreationJourney()
 
+  useEffect(() => {
+    if (stats != INITIAL_STATS_VALUE) {
+      setIsStatOpen(false)
+      setIsStatLoaded(false)
+      setStats(INITIAL_STATS_VALUE)
+    }
+  }, [offererId])
+
+  useEffect(() => {
```

Comment:
Je pense qu'on a probablement pas besoin de `useEffect` ici. Je ne connais pas cette page mais on tombe probablement dans un de ces 2 cas : 

- [Resetting all state when a prop changes](Adjusting some state when a prop changes )
- [Adjusting some state when a prop changes ](https://beta.reactjs.org/learn/you-might-not-need-an-effect#adjusting-some-state-when-a-prop-changes)

Reply:
Effectivement, corrigé !

Reply:
Je pense que ça peut s'appliquer aussi au useEffect qui dépend de `offererId` :slightly_smiling_face:

Reply:
Corrigé aussi !